src/git_config/mod.rs

```diff
@@ -64,11 +63,40 @@ impl GitConfig {
         }
     }
 
+    #[cfg(not(test))]
+    pub fn try_create_from_path(env: &DeltaEnv, path: &String) -> Option<Self> {
```

Comment:
WDYT about just returning `Self`? This function doesn't really return an `Option` since it never return `None`.

Reply:
Good point. Fixed.